README.md

```diff
@@ -1,3 +1,39 @@
-# uwa-historical-tour
 
-Tour the UWA campus online
+# UWA Historical Tour
+
+Take a 360º virtual tour of UWA's Winthrop Hall, Hacket Hall, Whelan Court and more!
+
+## Description
+### Look While Learning
+* Drag your mouse or finger around the screen to view your surroundings. 
+* Each image has information icons on points of interest. Hover over these icons for the name of the object; click it to learn its history!
+
+### Navigation
+> Experience the tour as though you are really walking around UWA's campus.
+* In each image, there are clickable navigation icons that will bring you to the location it is hovering over.
+* Additionally, there are navigation arrows in the bottom right corner that can be pressed to move to the next or previous location.
+* Click the Home button to return to the start of the tour! 
+
+### Map Orientation
+* At each point of the tour, the map in the bottom left corner will show your current location on the UWA campus with a noticeable persona icon.
+* Observe that the light blue circles on the map show the relative position of each point of the tour with one another.
+
+## Installation
+```bash
+yarn start:open
+(placeholder)
+```
+
+## Visuals
+```
+(placeholder)
+```
+
+## Authors & Acknowledgment
+Authors: ...(if you are comfortable with it)
+
+**The copying and use of any historical photos and text, unless otherwise stated, must fully acknowledge the UWA Archives intellectual property.** Please ensure that, in full, the P number and accompanying title of each image must be displayed with the photo. Refer to the document: **Hitorical_Photos.txt** for the P number and titles.
```

Comment:
```suggestion
**The copying and use of any historical photos and text, unless otherwise stated, must fully acknowledge the UWA Archives intellectual property.** Please ensure that, in full, the P number and accompanying title of each image must be displayed with the photo. Refer to the document: **Historical_Photos.txt** for the P number and titles.
```